packages/payment-detection/src/any/any-to-any-proxy-info-retriever.ts

```diff
@@ -3,16 +3,7 @@ import { PaymentTypes } from '@requestnetwork/types';
 import { BigNumber, ethers } from 'ethers';
```

Comment:
replace `any-to-erc20-proxy-info-retriever.ts` to a generic `any-to-any-proxy-info-retriever.ts`... called for `erc20conversion` AND `ethConversion`